libs/SalesforceSDKCommon/SalesforceSDKCommonTests/KeychainItemManagerTests.swift

```diff
@@ -26,37 +26,33 @@
 //  WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 import XCTest
-import SalesforceSDKCommon
+@testable import SalesforceSDKCommon
 
 final class KeychainItemManagerTests: XCTestCase {
     
     func testCreateIfNotPresent() {
         let accountName = "account.two"
         let serviceName = "test.two"
-        _  = KeychainHelper.remove(service: serviceName, account: accountName)
-        let keychainResult = KeychainHelper.createIfNotPresent(service: serviceName, account: accountName)
-        XCTAssertTrue(keychainResult.success)
-        
-        let keychainReadResult = KeychainHelper.read(service: serviceName, account: accountName)
-        XCTAssertTrue(keychainReadResult.success)
-        XCTAssertNil(keychainReadResult.data)
-        
-        let data = "ATESTSTRING2".data(using: .utf8)  ?? Data()
-        let writeResult = KeychainHelper.write(service: serviceName, data: data, account: accountName)
-        XCTAssertTrue(writeResult.success)
-        XCTAssertNotNil(writeResult.data)
-        
-        let readResult = KeychainHelper.read(service: serviceName, account: accountName)
+        var readResult = KeychainHelper.read(service: serviceName, account: nil)
```

Comment:
why is the account nil?

Reply:
It's a copy-and-paste error. Fixed, thanks!